lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -224,6 +242,32 @@ public S3FileSystemConfig setCannedAcl(ObjectCannedAcl objectCannedAcl)
         return this;
     }
 
+    public RetryMode getRetryMode()
```

Comment:
I don't fully follow why we would want to configure the retry mode.

Reply:
Should we change the default to be STANDARD? I wonder why they default to LEGACY.